ExampleApps/YOLOSingleImageSwiftUI/YOLOSingleImageSwiftUI/ContentView.swift

```diff
@@ -21,7 +21,7 @@ struct ContentView: View {
   @State private var inputImage: UIImage?
   @State private var yoloResult: YOLOResult?
 
-  let yolo = YOLO("yolo11n-seg", task: .segment)
+  let yolo = YOLO("yolo11s", task: .detect)
```

Comment:
@john-rocky we always want to default to the nano models for all examples. Should this be changed?